x/claim/types/codec.go

```diff
@@ -25,5 +25,14 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 }
 
 var (
-	ModuleCdc = codec.NewProtoCodec(cdctypes.NewInterfaceRegistry())
+	amino = codec.NewLegacyAmino()
+
+	// ModuleCdc = codec.NewProtoCodec(cdctypes.NewInterfaceRegistry())
+	ModuleCdc = codec.NewAminoCodec(amino)
```

Comment:
LGTM.

Could you check `x/alloc/types/codec.go` for consistency? We probably want to see the same code in both, right?